### ISSUE TITLE
Preprints: Add HTTP/S setting to Repository Wizard

### ISSUE THREAD

**Issue by alainna** [BirkbeckCTP/janeway#3307]
**Is your feature request related to a problem? Please describe.**
As a Janeway admin, I expect to be able to specify whether a new preprints repository should be HTTP or HTTPS during the initial setup, and to be able to later update this setting along with other settings.

**Describe the solution you'd like**
Add to the Janeway Repository Wizard a toggle box which corresponds to the "is secure" setting in Django Admin

![image](https://user-images.githubusercontent.com/15632391/205998813-02606d42-59a4-45bc-8b20-02f81f28a3cf.png)

![image](https://user-images.githubusercontent.com/15632391/205997984-c7a50d33-657e-4bfe-ae60-950a7805676b.png)

**Comment by alainna** [comment]
@ajrbyers true. Perhaps this should also/instead be an option via the manager? 

![image](https://user-images.githubusercontent.com/15632391/206000189-8e4d5b64-6b98-49d3-93c3-d4a4340683c7.png)

**Comment by ajrbyers** [comment]
> @ajrbyers true. Perhaps this should also/instead be an option via the manager?
> 
> ![image](https://user-images.githubusercontent.com/15632391/206000189-8e4d5b64-6b98-49d3-93c3-d4a4340683c7.png)

Oooo that’s good.
